### DISCUSSION TITLE
Names of selected labels are not displayed Multiselect

### DISCUSSION THREAD

**Discussion by ArtemBulavin** [filamentphp/filament#1436] [Help] [answered]
### Package

filament/filament

### Package Version

v2.9.16

### Laravel Version

v8.81.0

### Livewire Version

v2.10.1

### PHP Version

8.1

### Bug description

Names of selected items are not displayed in MultiSelect.
![image](https://user-images.githubusercontent.com/36136943/152101745-395ecfa6-b3c5-400c-a12c-58e4222ab9d7.png)
  Forms\Components\MultiSelect::make('similars')
                                    ->getSearchResultsUsing(fn (string $query) => Product::where('product_title', 'like', "%{$query}%")->pluck('product_title', 'id'))
                                    ->getOptionLabelsUsing(fn (array $values) => Product::find($values)->pluck('product_title'))
                                    ->label('Похожие товары'),

**Comment by MACscr** [comment]
should be ```Product::find($values)->pluck('product_title'))->toArray()```

**Comment by ArtemBulavin** [comment]
> should be `Product::find($values)->pluck('product_title'))->toArray()`

did not help(

**Comment by Nick-Kyle-Wonde** [comment, marked as answer]
This works for me.

`     ->getOptionLabelsUsing(function (array $values): array {
            return collect($values)->mapWithKeys(fn ($value) => [$value => Product::find($value)?->product_title])->toArray();
        });`

**Reply by ArtemBulavin** [reply]
Thank you very much, it worked!
